### PR TITLE
Implementation of the likes counter

### PR DESCRIPTION
**Pull request by ezequielsetzesglobant**
The likes counter was implemented.

Video:
https://user-images.githubusercontent.com/84338513/162813933-820b5be0-00b6-40d4-a8fb-3cd834550630.mov

Test:
Screen Shot 2022-04-11 at 16 17 04

### CODE REVIEW

File: lib/home_page.dart [hunk 1]
```diff
@@ -23,31 +30,65 @@ class HomePage extends StatelessWidget {
...
         ),
       ),
     );
   }
+
+  _incrementLikes() {
+    setState(() {
+      _likesCounter++;
+    });
+  }
```
Comment (MarcosCGlobant) [inline lib/home_page.dart]: Avoid defining method after the build(), please do it before

**Review by MarcosCGlobant — CHANGES_REQUESTED** [review]

File: lib/home_page.dart (hunk 1 above)
Comment (ezequielsetzesglobant) [reply lib/home_page.dart]: done

File: lib/home_page.dart [hunk 2]
```diff
@@ -23,27 +36,55 @@ class HomePage extends StatelessWidget {
...
+              const Padding(
                 padding: EdgeInsets.all(Constants.padding),
                 child: Text(
                   Constants.mail,
                   style: TextStyles.mailTextStyle,
                 ),
               ),
+              Padding(
+                padding: const EdgeInsets.all(Constants.padding),
+                child: Row(
```
Comment (Juliana-Dimeglio) [inline lib/home_page.dart:65]: You could change the right and left padding from this section and use the attribute of the Row named mainAxisAlignment: MainAxisAlignment to set a more dynamic position.

**Review by Juliana-Dimeglio — CHANGES_REQUESTED** [review]

File: lib/home_page.dart (hunk 2 above)
Comment (ezequielsetzesglobant) [reply lib/home_page.dart:65]: done

**Review by Juliana-Dimeglio — APPROVED** [review]

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Pull request merged**